repo2docker/buildpacks/nix/__init__.py

```diff
@@ -0,0 +1,70 @@
+"""BuildPack for nixpkgs environments"""
+import os
+
+from ..base import BaseImage
+
+
+class NixBuildPack(BaseImage):
```

Comment:
if we inherit from `PythonBuildPack` instead people will be able to use nix together with conda and pip.

This depends a bit on the use-case you see for nix. Should nix be something that people use without anything else (right now) or as an addition to install tools that they can't install with what they already have (a bit like `apt.txt` which lets you install some additional things as well as using your `environment.yml` or `REQUIRE`).